PowerEditor/installer/nativeLang/english.xml

```diff
@@ -954,7 +954,7 @@ Translation note:
 					<Item id="6106" name="Tab Bar"/>
 					<Item id="6107" name="Reduce"/>
 					<Item id="6108" name="Lock (no drag and drop)"/>
-					<Item id="6109" name="Darken inactive tabs"/>
+					<Item id="6109" name="Color background of inactive tabs"/>
```

Comment:
@alankilborn 
IMO - That's not the gist of the meaning here, that "darken" is used here more as the opposite of "highlight". At least that is what I use in my translation. I am not a native English speaker, so maybe my English attempts will not be ok, but I would use here expressions like "Fade inactive tab" or "Weaken inactive tab color" or simply "Dim inactive tabs".